extensions/amp-story/amp-story.md

```diff
@@ -254,53 +277,224 @@ The `bookend-config-src` value is a URL endpoint that returns data for the end s
   </amp-anim>
 </figure>
 
-#### Related articles
+To use it, include an `<amp-story-bookend>` tag as the child of your `<amp-story>` with the required attributes like so:
+
+```html
+<amp-story standalone>
+  <amp-story-page id="cover">
+    ...
+  </amp-story-page>
+  <!-- `src` and `layout=display` are required. -->
+  <amp-story-bookend src="bookendv1.json" layout=nodisplay>
+  </amp-story-bookend>
+<amp-story>
+```
+
+Next, you must create a JSON file where you can customize the bookend. The overall structure of the config looks like so:
+
+```text
+{
+  "bookend-version": "v1.0",
+  "share-providers": [
+    ...
+  ],
+  "components": [
+    ...
+  ]
+}
+
+```
+
+It is required to specify you are using the v1.0 version by including the first line.
+
+#### Bookend components
+
+The bookend is made up of a variety of components. These components can be articles, call to action links, text, and more.
+
+They are specified in the `components` field of the configured JSON. See the [Example JSON response](#example-json-response) section below for an example.
+
+##### heading
+
+The `heading` component has a `text` field, which can be used to append a title to a group of articles.
+
+
+```json
+{
+  "type": "heading",
+  "text": "More to Read"
+}
+```
+
+<amp-img alt="Bookend heading component" layout="fixed"
+src="https://github.com/ampproject/amphtml/raw/master/extensions/amp-story/img/amp-story-bookend-component-heading.png" width="386" height="123">
+  <noscript>
+    <img alt="Bookend heading component" src="img/amp-story-bookend-component-heading.png" />
+  </noscript>
+</amp-img>
+
+
+##### small
+
+The `small` component can be used to link to related articles. This component requires the following fields: `title`, `url`, and optionally an `image`.
+
+```json
+{
+  "type": "small",
+  "title": "This is India an the best places you should go",
+  "url": "http://example.com/article.html",
+  "image": "http://placehold.it/256x128"
+}
+```
+
+<amp-img alt="Bookend small component" layout="fixed"
+src="https://github.com/ampproject/amphtml/raw/master/extensions/amp-story/img/amp-story-bookend-component-small.png" width="379" height="192">
+  <noscript>
+    <img alt="Bookend small component" src="img/amp-story-bookend-component-small.png" />
+  </noscript>
+</amp-img>
+
+##### landscape
+
+The `landscape` component can be used for alternative formats of content, like videos. This component requires the following fields: `title`, `category`, `url`, and an `image`.
+
+```json
+{
+  "type": "landscape",
+  "title": "TRAPPIST-1 Planets May Still Be Wet Enough for Life",
+  "url": "http://example.com/article.html",
+  "category": "astronomy",
```

Comment:
Why is `category` required?  In some cases you might not want to provide a category, just a landscape image for another story. (same for portrait)

Reply:
I tried to stick to the mocks as much as possible and thought we wanted that in every landscape and portrait card. @hongwei1990 what do you think? Should we have the category as optional? (We call category the text on top of the title - 'ASTRONOMY' in the example below).

![image](https://user-images.githubusercontent.com/5449100/40850861-b8969452-6593-11e8-9d6e-537a362c31da.png)

Reply:
Problem I see is that if publishers don't specify neither a title nor category, we might end up with unexpected/undesirable results.

Reply:
But I would expect the same requirements we have for the "small" images, apply to all images - that is, a title, url, and image.  The only optional attribute is category.

Reply:
The elements here inside each card component is optional not required. If publisher doesn't specify eyebrow we will not show. Same logic apply to other elements.

Reply:
Sorry, correct my last comment. Title and URL are required.

Reply:
The validator says otherwise.  Let's make sure that expectations = validator = doc.

Reply:
Because of the JSON format, we can't actually check this in the AMP validator, but we can at least emit console errors at runtime, which @Enriqe will add.

Reply:
Okay, just to clarify and confirm, right now I am getting console errors on `portrait`: 

```
Portrait component must contain  category, image, and url fields..
```

I'm expecting:

- small:
  - required: title, url
  - optional:  image
- landscape
  - required: title, url, image
  - optional:  category
- portrait
  - required: url, image
  - optional:  category, title  (title is optional because it could be another story)